ampersand-multifield-view.js

```diff
@@ -73,9 +98,25 @@ var MultiFieldView = View.extend({
   },
 
   isValid: function() {
-    return this.fields.every(function(field) {
+    var fieldsValid =  this.fields.every(function(field) {
       return field.valid;
     });
+
+    var valid = !this.runTests();
+
+    return fieldsValid && valid;
+  },
+
+  runTests: function() {
+    var errMsg = '';
+
+    this.tests.some(function(test) {
+      errMsg = test.call(this, this.value) || '';
```

Comment:
Later, `this.message = errMsg` and it expects `this.message` to be a string. If there is no error, the test returns undefined or another falsey value.